test/parallel/test-async-hooks-async-await.js

```diff
@@ -0,0 +1,26 @@
+// Test async-hooks fired on right
+// asyncIds & triggerAsyncId for async-await
+'use strict';
+
+const common = require('../common');
```

Comment:
lint issue: the `common` is never used. Just make this...

```js
require('./common'); 
```

without assigning it to anything.

Reply:
@jasnell: Taken care also, rebased with master. So CI should be green now.